extensions/wikia/Insights/InsightsUnconvertedInfoboxesHooks.class.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+class InsightsUnconvertedInfoboxesHooks {
+	static public function onUnconvertedInfoboxesQueryRecached() {
+		F::app()->wg->Memc->delete( wfMemcKey( ApiQueryUnconvertedInfoboxes::MCACHE_KEY ) );
```

Comment:
I thought we are not using the F anymore and just $wgMemc->delete.. @owend What do you say?